pkg/ddevapp/craftcms.go

```diff
@@ -68,6 +69,14 @@ func craftCmsPostStartAction(app *DdevApp) error {
 		return nil
 	}
 
+	// Check version is v4 or higher or warn user about app type mismatch.
+	if !isCraftCms4orHigher(app) {
+		util.Warning("It looks like the installed Craft CMS is lower than version 4 where it's recommended to use project type `php` or disable settings management with `ddev config --disable-settings-management`")
+		if !util.Confirm("Would you like to continue anyway with the automatic configuration?") {
```

Comment:
I sure wish we had an option to offer "no" as the default. Should we do that?

Reply:
Or rephrase the question to invert it.

Reply:
That's not a bad idea.